core/bentley/src/BentleyError.ts

```diff
@@ -307,15 +307,6 @@ export enum IModelHubStatus {
   InitializationTimeout = IMODELHUBERROR_REQUESTERRORBASE + 7,
 }
 
-/** Authentication Errors
- * @beta Internal? Right package?
- */
-export enum AuthStatus {
```

Comment:
Can you put this removal in the NextVersion.md since we are dropping a beta API?